kolena/_experimental/object_detection/utils.py

```diff
@@ -208,8 +208,7 @@ def _compute_sklearn_arrays_by_class(
         y_true_by_label[label] = y_true
         y_score_by_label[label] = y_score
 
-    y_true, y_score = _compute_sklearn_arrays(all_matches)
-    return y_true, y_score, y_true_by_label, y_score_by_label
+    return y_true_by_label, y_score_by_label
```

Comment:
As one last non-blocking nit, this return type seems like it would be more ergonomic than the current one:

```python
Dict[str, Tuple[np.ndarray, np.ndarray]]
```